db/migrate/20221208104744_create_post.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
```

Comment:
- [ ] As previous reviewer suggested that text column should be **text** datatype and not **string**, please kindly fix that :upside_down_face: I can see that you have this text datatype in your schema.sql file but i didn't see any migration by which you have changed this **string** to **text**. Kindly make a migration for this and don't manually update the schema.sql file. Migrations are the blueprints of your database. Suppose you don't have a schema.sql file or it has been removed due to some reason. Then you can make your database using your migrations so that's why you need this migration :blush: